### ISSUE TITLE
Please update .gitignore to avoid checking in unnecessary files

### ISSUE THREAD

**Issue by XiaoYuClassroom** [VinceG/twitter-bootstrap-wizard#125]
Thanks for the plugin. Great work!

I totally understand your pain in entertaining all kinds of package managers and intention to distribute this package through GitHub master branch. But please add some filters in your .gitignore file so that no redundant files are checked in.

For example,
*~
.DS_Store*

Thanks.

**Comment by jamiesonbecker** [comment]
Good point, and this is a great package. I'd add the minified prettify.* to .gitignore or move them into the examples as well, since they're only (?) for the examples to highlight the HTML.
